docs/en/contributor/Contributor-Getting-Started.md

```diff
@@ -294,6 +294,31 @@ Congratulations! You have successfully contributed to Alluxio! Thank you for joi
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/QsbM804rc6Y" frameborder="0" allowfullscreen></iframe>
 
+## Submit Your Feature
+
+If you have a brilliant idea about a new Alluxio feature, we highly encourage you to implement it and 
+contribute to the Alluxio repository.
+
+1. Create a Github issue in [Alluxio repository](https://github.com/Alluxio/alluxio/issues) regarding your feature.
+1. Attach a feature design document following the [template]({{ '/templates/Design-Document-Template.md' | relativize_url }})
+in your Github issue. The design document should follow the template and provide information regarding each section.
+We recommend using a public Google doc for more efficient collaboration and discussions.
+If a Google doc is not an option for you, you can choose to attach either a Markdown file or a PDF file for the design document.
+1. Tag your issue `type-feature`. Alluxio members periodically check all open issues and will allocate reviewers to the design document.
+Community users do not have the access to assign issues in Alluxio repository. So if you would like to initiate the review process,
+you can create a placeholder pull request, link to the issue and attach the feature design document, as request reviews from Alluxio members.
+The feature reviews and discussions will then be moved the placeholder pull request as an alternative.
```

Comment:
I don't quite understand this line

Reply:
Hmmm I tried to rephrase the expression.